__tests__/integration/configuration/test.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   consoleLevel: process.env.TEST_CONSOLE_LEVEL || 'silly',
 
   // Defines how long tests should wait before consulting balances after transactions
-  wsUpdateDelay: process.env.TEST_WS_UPDATE_DELAY || 1000,
+  wsUpdateDelay: process.env.TEST_WS_UPDATE_DELAY || 2000,
```

Comment:
I was getting error in the multisig integration test when running in the CI. The error was out of funds to send a transaction, which indicated that the WS update didn't arrive before running the next test. 

After this test, everything succeeded but this shouldn't be considered a final solution, the integration tests deserve a bigger refactor.

Reply:
Two considerations on this:
1) A change with less impact on the overall test duration would be to add manual delays on the tests that are failing the most ( the ones on MultiSig ). But given this error is happening more often, I agree changing the config is also a valid solution.

2) The best solution for this issue I believe would be using the `external notifications` feature, that was implemented after the integration tests were built. With it, an event would be triggered when the WS update was completed, avoiding the need for arbitrary delay periods.

Reply:
I will create an issue regarding this problem and describe possible solutions.